### PR TITLE
NDRS-456: update accounts.csv to use new hex-formatted public key which includes algorithm tag

### PR DESCRIPTION
**Pull request by Fraser999**

### CODE REVIEW

File: node/src/components/chainspec_loader/chainspec.rs [hunk 1]
```diff
@@ -130,25 +130,21 @@ impl Loadable for Vec<GenesisAccount> {
     fn from_file<P: AsRef<Path>>(path: P) -> Result<Self, Self::Error> {
         #[derive(Debug, Deserialize)]
         struct ParsedAccount {
-            public_key: String,
-            algorithm: String,
-            balance: String,
-            bonded_amount: String,
+            public_key: PublicKey,
+            balance: U512,
+            bonded_amount: U512,
```
Comment (sacherjj) [inline node/src/components/chainspec_loader/chainspec.rs:135]: Should this also have a u8 for algorithm?  This looks like it lost the algorithm.

Comment (dwerner) [reply node/src/components/chainspec_loader/chainspec.rs:135]: I _believe_ from the changes to accounts.csv this is intended to be derived from the key itself now - as the type is prefixed onto the key (the additional `01` at the start of each). `algorithm` would then be ignored by serde.

**Review by dwerner — APPROVED** [review]

File: node/src/components/chainspec_loader/chainspec.rs (hunk 1 above)
Comment (Fraser999) [reply node/src/components/chainspec_loader/chainspec.rs:135]: The algorithm is identified by the first two characters of the hex bytes (you'll see the accounts.csv files have been updated to include that info).  So the `PublicKey` type is an enum which can be parsed directly from a hex-encoded string.

**Review by sacherjj — APPROVED** [review]

**Comment by goral09** [comment]
I actually preferred the original version but it doesn't matter now.

**Comment by momipsl** [comment]
Have updated the `nctl-assets-setup` command accordingly.

**Pull request merged**